design-tokens/tokens/layer1/base.json

```diff
@@ -1,109 +1,129 @@
-        "type": "spacing"
+      "400": {
+        "value": "2rem",
+        "type": "dimension"
       },
-      "multi-value": {
-        "value": "{size.dimension.sm} {size.dimension.xl}",
-        "type": "spacing",
-        "description": "You can have multiple values in a single spacing token. Read more on these: https://docs.tokens.studio/available-tokens/spacing-tokens#multi-value-spacing-tokens"
-      }
-    },
-    "borderRadius": {
-      "sm": {
-        "value": "4",
-        "type": "borderRadius"
+      "500": {
+        "value": "2.5rem",
+        "type": "dimension"
       },
-      "lg": {
-        "value": "8",
-        "type": "borderRadius"
+      "600": {
+        "value": "3rem",
+        "type": "dimension"
       },
-      "xl": {
-        "value": "16",
-        "type": "borderRadius"
+      "800": {
+        "value": "4rem",
+        "type": "dimension"
+      }
+    },
+    "sizes": {
+      "global": {
+        "xs": {
+          "value": "0.25rem",
+          "type": "dimension"
+        }
       },
-      "multi-value": {
-        "value": "{size.borderRadius.sm} {size.borderRadius.lg}",
-        "type": "borderRadius",
-        "description": "You can have multiple values in a single radius token. Read more on these: https://docs.tokens.studio/available-tokens/border-radius-tokens#single--multiple-values"
+      "icons": {
+        "sm": {
+          "value": "1rem",
+          "type": "dimension"
+        },
+        "md": {
+          "value": "1.25rem",
+          "type": "dimension"
+        },
+        "lg": {
+          "value": "1.5rem",
+          "type": "dimension"
+        }
       }
     }
   },
-  "opacity": {
-    "low": {
-      "value": "10%",
-      "type": "opacity"
+  "borderRadius": {
+    "sm": {
+      "value": "2",
+      "type": "borderRadius"
+    },
+    "lg": {
+      "value": "8",
+      "type": "borderRadius"
     },
     "md": {
-      "value": "50%",
-      "type": "opacity"
+      "value": "4",
+      "type": "borderRadius"
+    },
+    "none": {
+      "value": "0",
+      "type": "borderRadius"
     },
-    "high": {
-      "value": "90%",
-      "type": "opacity"
+    "circle": {
+      "value": "700",
+      "type": "borderRadius"
     }
   },
-  "fontFamilies": {
-    "heading": {
+  "fontFamily": {
+    "mono": {
+      "value": "Ubuntu Mono",
+      "type": "fontFamilies"
+    },
+    "sans": {
       "value": "Inter",
       "type": "fontFamilies"
     },
-    "body": {
-      "value": "Roboto",
+    "fallbackMono": {
+      "value": "ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, \"Liberation Mono\", \"Courier New\", monospace",
+      "type": "fontFamilies"
+    },
+    "fallbackSans": {
+      "value": "ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, \"Helvetica Neue\", Arial, \"Noto Sans\", sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\", \"Noto Color Emoji\"",
       "type": "fontFamilies"
     }
```

Comment:
Ok I see, we will have to see how we can work with that in the lib and the typography components. There might be some changes in the future.